operators/advanced_network/managers/dpdk/adv_network_dpdk_mgr.cpp

```diff
@@ -49,9 +50,12 @@ struct RxWorkerParams {
   int num_segs;
   uint32_t batch_size;
   struct rte_ring* ring;
+  struct rte_ring* lb_ring;
   struct rte_mempool* flowid_pool;
-  struct rte_mempool* burst_pool;
-  struct rte_mempool* meta_pool;
+  struct rte_mempool* rx_burst_pool;
+  struct rte_mempool* tx_burst_pool;
+  struct rte_mempool* rx_meta_pool;  
+  struct rte_mempool* tx_meta_pool;
```

Comment:
sorry if this is too early as you're clearly still in draft, but if you can eventually add an inline comment for all of them for a short description of their role that'd be great, especially noting that so many of them are exclusively for SW loopback which isn't relevant once running properly.